data/json/items/ammo/40x46mm.json

```diff
@@ -87,7 +87,7 @@
     "id": "40x46mm_slug_m118",
     "copy-from": "40x46mm_grenade",
     "type": "AMMO",
-    "name": { "str": "40x46mm-M118 slug, reloaded", "str_pl": "40x46mm-M118 slugs, reloaded" },
+    "name": { "str_sp": "40x46mm-M118 slug, reloaded" },
```

Comment:
Ones with "slug" - something to be checked on, although if it indeed is never shown as plural, it's fine. Otherwise, "slug" is one of the terms like "round" and "flechette" that do need to be altered if plural - unlike, say, "##mm FMJ", which is both singular and plural.